treeherder/webapp/api/refdata.py

```diff
@@ -46,6 +49,17 @@ class RepositoryViewSet(CacheResponseAndETAGMixin,
     def list_cache_key_func(self, **kwargs):
         return models.REPOSITORY_LIST_CACHE_KEY
 
+    """
+    Overrides the retrieve method to get the extra information from the Jobs model
+    """
+    def retrieve(self, request, *args, **kwargs):
+        request = th_serializers.RepositorySerializer(self.queryset.get(pk=kwargs['pk']))
+        new_request = request.data.copy()
+        new_dict = json.loads(project_info(self.queryset, request.data['name']).content)
```

Comment:
Instead of doing this, you should just copy the relevant code from projects.py (in the same directory) and use that to update the dictionary.

    with JobsModel(request.data['name']) as jobs_model:
         new_request.update({'max_job_id': jobs_model.get_max_job_id()})

(not sure if that's exactly right but you get the idea)

You should also delete projects.py and update urls.py as discussed in the original bug.